### PR TITLE
Add example code for App Distribution Alert (New iOS Tester Device)

### PR DESCRIPTION
**Pull request by kaibolay**

### CODE REVIEW

**Review by jfcong — APPROVED** [review]

File: 2nd-gen/alerts-to-discord/functions/index.js
Comment (jfcong) [inline 2nd-gen/alerts-to-discord/functions/index.js:114]: Is App Distro alert app specific? It is important for users to know which app the new device is registered for?

Comment (kaibolay) [reply 2nd-gen/alerts-to-discord/functions/index.js:114]: Yes it is. Good catch! The Crashlytics is also app specific. I've added the `appId` to both messages.

**Review by colerogers — APPROVED** [review]
lgtm, I'd get a sign off from jeff or eric before merging

File: 2nd-gen/alerts-to-discord/functions/index.js
Comment (jhuleatt) [inline 2nd-gen/alerts-to-discord/functions/index.js:120]: Developers will copy and paste this code - do we want to encourage them to share this much tester info into a team chat room? Would this sample still be useful/realistic if we only include `testerDeviceModelName` and `appId`, or maybe `testerName` as well?

**Review by jhuleatt — APPROVED** [review]

**Comment by jhuleatt** [comment]
Thanks @kaibolay! Please do not merge until a @VictoriaAuto or @egilmorez have approved the includecode snippets in this PR

File: 2nd-gen/alerts-to-discord/functions/index.js
Comment (kaibolay) [reply 2nd-gen/alerts-to-discord/functions/index.js:120]: Good question! 

The message is really only actionable for developers who can update the provisioning profile and re-build and re-distribute an iOS app. So I'd expect this to be used to post to chat rooms with an appropriately limited audience. For other people the message would just be spam. For updating the provisioning profile, the developers need at least the App ID and the UDID (`${testerDeviceIdentifier}`) - the latter being the most sensitive value. So in order to be useful the message needs that data...

Or the example code gets modified to not post to Discord at all, but gets used as a basis to integrate with a CI/CD pipeline.

So I think it makes sense to extract *all* values from the payload just to demonstrate which values are available.

File: 2nd-gen/alerts-to-discord/functions/index.js
Comment (VictoriaAuto) [inline 2nd-gen/alerts-to-discord/functions/index.js:102]: Add a link to product docs: Firebase Alerts triggers (https://firebase.google.com/docs/functions/beta/alert-events) and Register additional iOS devices (https://firebase.google.com/docs/app-distribution/register-additional-devices#trigger-function-new-udids). Note that Eventarc updates to these docs will be published on Oct. 18, so you might want to wait until then to add these links here.

File: 2nd-gen/alerts-to-discord/functions/index.js
Comment (VictoriaAuto) [inline 2nd-gen/alerts-to-discord/functions/index.js:101]: Add space between "App" and "Distribution"

**Review by VictoriaAuto — CHANGES_REQUESTED** [review]
Left one nit comment and a comment to link to product docs when they're published. Let me know if you have any questions.

File: 2nd-gen/alerts-to-discord/functions/index.js
Comment (kaibolay) [reply 2nd-gen/alerts-to-discord/functions/index.js:102]: Yes, let's do that in a follow-up PR when those links exist.

File: 2nd-gen/alerts-to-discord/functions/index.js
Comment (kaibolay) [reply 2nd-gen/alerts-to-discord/functions/index.js]: Done.

**Review by VictoriaAuto — APPROVED** [review]

**Pull request merged**
